deps/chakrashim/lib/chakra_shim.js

```diff
@@ -214,12 +267,20 @@
       isPrepared = true;
       // Notify original Error object of this setter call. Without knowing
       // this Chakra runtime would reset stack at throw time.
-      Reflect_apply(oldStackDesc.set, e, [value]);
+      Reflect_apply(oldStackDesc.set, err, [value]);
```

Comment:
yeah, after I added `Reflect_apply(oldStackDesc.set, err, ['']);` below, we don't need this change. I will remove it.